### ISSUE TITLE
GCP Peering endless terraform apply

### ISSUE THREAD

**Issue by rrondeau** [mongodb/terraform-provider-mongodbatlas#41]
Hi

I cant get this provider to work with peering.
I start with an empty project, no container.

This is my code :
```
data "mongodbatlas_project" "test" {
  name = "test-project"
}

data "google_compute_network" "network" {
  project = var.project
  name    = var.network
}

resource "mongodbatlas_network_container" "test" {
  project_id       = data.mongodbatlas_project.test.id
  atlas_cidr_block = "10.1.0.0/18"
  provider_name    = "GCP"
}

resource "mongodbatlas_network_peering" "test" {
  project_id     = data.mongodbatlas_project.test.id
  container_id   = mongodbatlas_network_container.test.container_id
  provider_name  = "GCP"
  network_name   = data.google_compute_network.network.name
  gcp_project_id = var.project
}

resource "google_compute_network_peering" "atlas_peer" {
  name         = "atlas-peering"
  network      = data.google_compute_network.network.self_link
  peer_network = "https://www.googleapis.com/compute/v1/projects/${mongodbatlas_network_container.test.gcp_project_id}/global/networks/${mongodbatlas_network_container.test.network_name}"
}
```

The thing is `mongodbatlas_network_peering` wait for the peering to be up but on the other end `mongodbatlas_network_container` does not return the values to the `google_compute_network_peering` resource.

What did i do wrong ?

**Comment by rrondeau** [comment]
The peering is stuck on `WAITING_FOR_USER` status.
It appears that this status is considered like pending in the source : https://github.com/terraform-providers/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_network_peering.go#L247

**Comment by rrondeau** [comment]
I manage to get it working with this steps :
  - stop the run
  - find the peering id
  - import the peering in the pending state
  - re-apply to create the remaining gcp peering side of things

This is of course a temporary solution, can you point me the right solution/example ?

**Comment by themantissa** [comment]
@rrondeau  thank you, we are working on this issue.  @PacoDw - fyi.

**Comment by PacoDw** [comment]
Thank you so much for your review! it's so helpful 👍

**Comment by PacoDw** [comment]
Hello @rrondeau we have made the fix on the PR #43, could you check the example configuration on there and test on the `Network peering` branch? 

If you have another comment plz let us know, thanks!

**Comment by rrondeau** [comment]
Hello @PacoDw , it works like a charm ! Thank you !
